### DISCUSSION TITLE
roadiehq/backstage-plugin-aws-lambda

### DISCUSSION THREAD

**Discussion by 3200646** [RoadieHQ/roadie-backstage-plugins#920] [Q&A]
Hello , Getting error while trying to integrate roadie plugin with Backstage , roadiehq/backstage-plugin-aws-lambda 

<img width="906" alt="AWS_lambda_Error" src="https://user-images.githubusercontent.com/126150680/228829419-c534d937-de68-45ab-aa6c-93b77af5a051.png">

**Comment by Satya7340** [comment]
@kissmikijr, could you please help?
